### ISSUE TITLE
Adding Interest/Groups to lists

### ISSUE THREAD

**Issue by nikkithecrazyazn** [shahriarhossain/MailChimp.Api.Net#20]
How do you add interests/groups to a list so that it can later be segmented for a mailing??

**Comment by enkodellc** [comment]
There is some methods built into the wrapper for the interest categories bur some are not exposed or finished. I will try and add them if you want to test them out?

**Comment by MrSethT** [comment]
Yes, please do this.  I have been switching from library to library trying to find one that will do this, but haven't found one yet.  I thought I could do it in this one using SegmentOpts.Conditions, but they only accept integer "values".  I am willing to test/help any way I can.
